x-pack/elastic-agent/pkg/remote/client.go

```diff
@@ -142,7 +139,7 @@ func NewWithConfig(log *logger.Logger, cfg Config, wrapper wrapperFunc) (*Client
 
 		httpClient := http.Client{
```

Comment:
Does this need to be `http2.Client`? Or will the `httpcommon.WithForceAttepmtHTTP2(true)` switch it to http2 transparently?

Reply:
the net/http package can upgrade to http2. HTTP 1 or 2 is handled by the transport.